### PR TITLE
Quantum volume: num_circuits vs repetitions

### PR DESCRIPTION
**Pull request by mpharrigan**
"repetitions" refer to running the same circuit a bunch of times.
"num_circuits" is how many circuits you use to estimate quantum
volume / HOG.

### CODE REVIEW

File: cirq/contrib/quantum_volume/quantum_volume.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def compute_heavy_set(circuit: cirq.Circuit) -> List[int]:
 def sample_heavy_set(circuit: cirq.Circuit,
                      heavy_set: List[int],
                      *,
-                     repetitions=10000,
+                     repetitions=10_000,
```
Comment (Strilanc) [inline cirq/contrib/quantum_volume/quantum_volume.py]: I like it.

File: cirq/contrib/quantum_volume/quantum_volume.py [hunk 2]
```diff
@@ -207,23 +207,23 @@ def prepare_circuits(
...
+        num_circuits: The number of circuits to create.
         random_state: A way to seed the RandomState.
 
     Returns:
         A list of tuples where the first element is a generated model
         circuit and the second element is the heavy set for that circuit.
     """
     circuits = []
-    for repetition in range(num_repetitions):
+    for repetition in range(num_circuits):
```
Comment (Strilanc) [inline cirq/contrib/quantum_volume/quantum_volume.py]: Rename the iteration variable too.

File: cirq/contrib/quantum_volume/quantum_volume.py (hunk 1 above)
Comment (mpharrigan) [reply cirq/contrib/quantum_volume/quantum_volume.py]: hashtag feature I floated in python-dev irc room a couple years ago and everyone shouted at me for saying it's dumb before implementing it a couple releases later

File: cirq/contrib/quantum_volume/quantum_volume.py (hunk 2 above)
Comment (mpharrigan) [reply cirq/contrib/quantum_volume/quantum_volume.py]: good catch. updated docstrings as well

**Comment by mpharrigan** [comment]
cc @KevinVillela

**Comment by KevinVillela** [comment]
Thanks for doing this. I believe you may also need to update https://github.com/quantumlib/Cirq/blob/master/examples/advanced/quantum_volume.py.

**Comment by mpharrigan** [comment]
updated that file too

File: examples/advanced/quantum_volume.py [hunk 3]
```diff
@@ -68,7 +68,7 @@ def parse_arguments(args):
                         type=int,
                         help='Seed for the Random Number Generator.')
     parser.add_argument(
-        '--num_repetitions',
+        '--num_circuits',
         default=100,
         type=int,
         help='The number of times to run the circuit on the quantum computer.'
```
Comment (KevinVillela) [inline examples/advanced/quantum_volume.py]: This doc should now say something like "The number of circuits to generate". Or, to not reproduce the message, "Pass-through to the Quantum Volume module".

**Pull request merged**